word_guess.rb

```diff
@@ -0,0 +1,269 @@
+require 'colorize'
+
+class WordGuess
+
+  # creates a game object that contains all the game methods
+  # sets the word
+  def initialize
+    @guess = "" # initializes w/ empty guess
+    @wrong_count = 5 # Maybe will change this difficulty level
+    @difficulty = "" # init with empty difficulty so player can choose
+    @pick_word = {easy: ["dog", "cat", "star"].sample,
+      medium: ["pizza", "input", "coder"].sample,
+      hard: ["birthday", "computation", "ubiquitous"].sample} # selects a word for each difficulty
+    @word = "" # placeholder for word we will be guessing
+    @alphabet_array = ("A".."Z").to_a # letters to check guesses against
+    @used_letters = [] # place holder for guessed letters
+    @word_so_far = "" # placeholder to hold the word as it is guessed
+    @bad_prize = ["stinky socks", "moldy tofu", "rotten eggs", "more homework", "dirty diapers"].sample
+    @good_prize = ["elephant hotdog", "invisible pineapple", "expensive whiskey", "customized puppet of yourself"].sample
+  end
+
+
+  # this method enables playing the game
+  def play_the_game
+    pick_a_word
+    update_board
+    until @wrong_count == 0
+      guess
+    end
+    lose_game
+  end
+
+  private
+
+  # this method defines the difficulty level and sets the word to guess
+  def pick_a_word
+    if @word == ""
+      puts "What difficulty would you like to play at? There's easy(E), medium(M), or hard(H)!"
+      print "Difficulty: "
+      @difficulty = gets.chomp.upcase
+      if @difficulty == "E" || @difficulty == "EASY"
+        @word = @pick_word[:easy]
+        @word = @word.upcase
+      elsif @difficulty == "M" || @difficulty == "MEDIUM"
+        @word = @pick_word[:medium]
+        @word = @word.upcase
+      elsif @difficulty == "H" || @difficulty == "HARD"
+        @word = @pick_word[:hard]
+        @word = @word.upcase
```

Comment:
Since this line is the same for all branches of this conditional, you could move it to the end of this section right before you set the `@word_so_far` variable. This would remove the duplication within each of these conditionals.